### PR TITLE
Update vignettes

### PR DESCRIPTION
**Pull request by PavelBal**
- Update vignettes
- `DataMapping` is clonable

### CODE REVIEW

File: R/esqlabsLegacy-env.R [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.getPackageVersion <- function() {
```
Comment (PavelBal) [inline R/esqlabsLegacy-env.R:1]: This file was missing

**Comment by PavelBal** [comment]
Just added an up-to-date vignette on figure creation.

File: vignettes/introduction-esqlabsr.Rmd [hunk 2]
```diff
@@ -28,10 +28,8 @@ In order to load a simulation in R, it must be present in the **\*.pkml** file f
 
 The documentation is divided into three parts covering the following aspects:
```
Comment (IndrajeetPatil) [inline vignettes/introduction-esqlabsr.Rmd]: Is it still three parts?

**Review by IndrajeetPatil — APPROVED** [review]
Looks good.

Can you please style also the code in Rmd:

```r
styler::style_dir(filetype = c("R", "Rmd"))
```

File: vignettes/introduction-esqlabsr.Rmd (hunk 2 above)
Comment (PavelBal) [reply vignettes/introduction-esqlabsr.Rmd]: Ups

**Pull request merged**